pkg/datapath/iptables/iptables.go

```diff
@@ -62,6 +63,23 @@ const (
 var (
 	isWaitMinVersion        = versioncheck.MustCompile(">=1.4.20")
 	isWaitSecondsMinVersion = versioncheck.MustCompile(">=1.4.22")
+	dnsPorts                = func(port string) ([]*lb.L4Addr, error) {
```

Comment:
I'm wondering why this function is called `**dns**Ports`? From the annotation doc above, it seems that the NOTRACK rule can be applied not only for DNS ports?

Reply:
Renamed to `noTrackPorts`.